### PR TITLE
Maven publish changes

### PR DESCRIPTION
**Pull request by arindamhit**
The changes we made in the build.gradle files to publish the artifacts in the maven

### CODE REVIEW

File: mas-foundation/build.gradle [hunk 1]
```diff
@@ -206,6 +153,85 @@ artifacts {
...
+                    }
+                    issueManagement {
+                        system = 'github'
+                        url = issueUrl
+                    }
+                    developers {
+                        developer {
+                            id = 'apigatewayopensource'
+                            name = 'CA API Gateway Bot'
+                            email = 'API-Gateway-OpenSource@ca.com'
```
Comment (graju256) [inline mas-foundation/build.gradle:206]: is this still valid?

File: mas-foundation/build.gradle [hunk 2]
```diff
@@ -206,6 +153,85 @@ artifacts {
     archives sourcesJar
 }
 
+afterEvaluate {
```
Comment (graju256) [inline mas-foundation/build.gradle:156]: I assume, we should do the same for other modules in this repo. Don't we?

File: mas-foundation/build.gradle [hunk 3]
```diff
@@ -137,60 +135,9 @@ task javadoc(type: Javadoc) {
     failOnError false
 }
 
-//jcenter integration
 group 'com.ca'
 
 project.version = mas_version
-bintray {
```
Comment (graju256) [inline mas-foundation/build.gradle:144]: I assume, we should do the same for other modules in this repo. Don't we?

File: mas-foundation/build.gradle (hunk 1 above)
Comment (arindamhit) [reply mas-foundation/build.gradle:206]: This part has to be validated with Scott. Will conclude on this.

File: mas-foundation/build.gradle (hunk 3 above)
Comment (arindamhit) [reply mas-foundation/build.gradle:144]: yes.. we are going to do the same for other repos.

File: mas-foundation/build.gradle (hunk 2 above)
Comment (arindamhit) [reply mas-foundation/build.gradle:156]: yes.. we are going to do the same for other repos.

File: gradle.properties [hunk 4]
```diff
@@ -33,5 +33,25 @@ mas_version=
 siteUrl = 'https://www.ca.com/us/developers/mas.html'
 gitUrl = 'https://github.com/CAAPIM/Android-MAS-SDK.git'
 issueUrl = 'https://github.com/CAAPIM/Android-MAS-SDK/issues'
+PUBLISH_GROUP_ID = 'com.ca.apim'
+version = '2.2'
+artifactId = '0.0'
```
Comment (scottchc1) [inline gradle.properties:38]: you set artifactId in each module's build config. what is this var in properties for?

Comment (arindamhit) [reply gradle.properties]: I just removed that..

File: mas-storage/build.gradle [hunk 5]
```diff
@@ -100,74 +100,101 @@ task javadoc(type: Javadoc) {
     include '**/*MAS*.java'
 }
 
-//jcenter integration
 group 'com.ca'
```
Comment (scottchc1) [inline mas-storage/build.gradle:103]: is "group" var still used by anything? it's unclear the actual effect of
`group` vs.
`PUBLISH_GROUP_ID` (in gradle.properties) vs.
`groupId`

File: mas-foundation/build.gradle (hunk 1 above)
Comment (scottchc1) [reply mas-foundation/build.gradle:204]: i can't answer this. i sent question to Product team to verify. if they don't respond in time for your merge, i would say keep status quo and update later

**Review by ysd24 — APPROVED** [review]

**Pull request merged**